src/providers/ASAModuleUpdateCodeLensProvider.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
How about put the provider into intellisense folder?

Reply:
OK